mezzanine/core/templatetags/mezzanine_tags.py

```diff
@@ -429,12 +429,17 @@ def thumbnail(
             pad_container = Image.new("RGBA", pad_size, padding_color)
             pad_container.paste(image, (pad_left, pad_top))
             image = pad_container
+            # Make thumbnail a PNG - required if original isn't one
```

Comment:
I think this change probably doesn't need to go back to main, I'm sure there's a better solution to changing a JPG's thumbnail to PNG (to allow for transparency background when using padding). Feel free to skip these lines on merge